Frontend/fiscales-app-ts/.gitignore

```
@@ -19,6 +19,7 @@
 # misc
 .DS_Store
 *.pem
+*.idea/
```

Sugiero agregar acá el archivo .env para evitar que alguien suba información confidencial
```suggestion
*.idea/
.env
```

---

en Next.js se usa el `.env.local` para poner secrets q no deber subirse. y ya esta incluido en el `.gitignore`.

---

entiendo tu punto @navarroaxel de todos modos nada que no se utilice debe estar en el repositorio, en el comentario anterior comparti los pasos para eliminar el archivo